.code-samples.meilisearch.yaml

```
@@ -733,3 +740,29 @@ tenant_token_guide_generate_sdk_1: |-
 tenant_token_guide_search_sdk_1: |-
   Client frontEndClient = new Client(new Config("http://localhost:7700", token));
   frontEndClient.index("patient_medical_records").search("blood test");
+date_guide_index_1: |-
+  import com.meilisearch.sdk;
+  import org.json.JSONArray;
+  import java.nio.file.Files;
+  import java.nio.file.Path;
+
+  Path fileName = Path.of("games.json");
+  String gamesJson = Files.readString(fileName);
+  Index index = client.index("games");
+  index.addDocuments(gamesJson);
+date_guide_filterable_attributes_1: |-
+  client.index("movies").updateFilterableAttributesSettings(new String[]
+  {
+    "release_timestamp"
+  });
```

One liner it will be better in this case
```suggestion
  client.index("movies").updateFilterableAttributesSettings(new String[] { "release_timestamp" });
```